HEC.FDA.ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -42,11 +44,15 @@ public GriddedImporterVM(EditorActionManager actionManager) : base(actionManager
         /// <summary>
         /// Constructor used when editing an existing child node.
         /// </summary>
-        /// <param name="elem"></param>
-        /// <param name="actionManager"></param>
         public GriddedImporterVM(HydraulicElement elem, EditorActionManager actionManager) : base(elem, actionManager)
         {
-            SelectedPath = Connection.Instance.HydraulicsDirectory + "\\" + elem.Name;
+            IEnumerable<IHydraulicProfile> profiles = elem.DataSet.HydraulicProfiles.OrderBy(x => x.Probability);
+            ListOfRows.Clear();
+            foreach (IHydraulicProfile profile in profiles)
```

Comment:
This was the source of our bug. I don't know why I thought the prior would work. That was definitely my doing. now we're actually loading the profiles from the element.